spec/src/main/asciidoc/architecture.adoc

```diff
@@ -243,8 +243,8 @@ In the above examples both `countMeA()` and `countMeB()` will share a single Cou
 [[cdi_scopes]]
 ==== Metrics and CDI scopes
 
-All metric types except gauges support being declared using annotations in CDI beans which use such scope that there will be multiple instances of the bean created 
-over the lifecycle of the application. If multiple instances of the bean with a metric exist, usages of the metric will be merged together, so for example,
+All metric types except gauges support being declared using annotations in CDI beans.  Depending on the CDI bean scope there may be multiple instances of the bean created over the lifecycle of the application.
```

Comment:
The first sentence isn't correct, you can declare a gauge by annotation in a CDI bean, there's just the limitation that the bean must be of such scope that there will be only one instance.

Reply:
thanks @jmartisk -- I missed that subtlety before.  I've added a commit to fix that.